### PR TITLE
Split linux installers

### PR DESCRIPTION
**Pull request by nirajpathak13**
Make individual workflows for building .rpm and .deb packages

### CODE REVIEW

File: .github/workflows/build-linux-installer-rpm.yml [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+    - name: Create Checksums
+      if: startsWith(github.ref, 'refs/tags/')
+      env:
+        CHIA_INSTALLER_VERSION: ${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}
+      run: |
+         ls ${{ github.workspace }}/build_scripts/final_installer/
+         sha256sum ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain-${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}-1.x86_64.rpm > ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain-${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}-1.x86_64.rpm.sha256
+         ls ${{ github.workspace }}/build_scripts/final_installer/
+#         sha256sum ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}_amd64.deb > ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}_amd64.deb.sha256
```
Comment (mishan) [inline .github/workflows/build-linux-installer-rpm.yml:144]: nit: remove commented lines in this file pertaining to deb and commented lines in the other file pertaining to rpm?

Comment (nirajpathak13) [reply .github/workflows/build-linux-installer-rpm.yml:144]: Thanks. This is still a work in progress and not ready for review yet.

File: .github/workflows/build-linux-installer-rpm.yml [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+      if: startsWith(github.ref, 'refs/tags/')
+      id: tag-name
+      run: |
+          echo "::set-output name=TAG_NAME::$(echo ${{ github.ref }} | cut -d'/' -f 3)"
+          echo "::set-output name=REPO_NAME::$(echo ${{ github.repository }} | cut -d'/' -f 2)"
+
+    - name: Mark installer complete
+      if: startsWith(github.ref, 'refs/tags/')
+      run: |
+          curl -s -XPOST -H "Authorization: Bearer ${{ secrets.GLUE_ACCESS_TOKEN }}" --data '{"chia_ref": "${{ steps.tag-name.outputs.TAG_NAME }}"}' ${{ secrets.GLUE_API_URL }}/api/v1/${{ steps.tag-name.outputs.REPO_NAME }}/${{ steps.tag-name.outputs.TAG_NAME }}/success/build-linux
```
Comment (cmmarslender) [inline .github/workflows/build-linux-installer-rpm.yml:172]:
```suggestion
          curl -s -XPOST -H "Authorization: Bearer ${{ secrets.GLUE_ACCESS_TOKEN }}" --data '{"chia_ref": "${{ steps.tag-name.outputs.TAG_NAME }}"}' ${{ secrets.GLUE_API_URL }}/api/v1/${{ steps.tag-name.outputs.REPO_NAME }}/${{ steps.tag-name.outputs.TAG_NAME }}/success/build-linux-rpm
```

Comment (cmmarslender) [reply .github/workflows/build-linux-installer-rpm.yml:172]: Also need to change the other one to build-linux-deb or something as well

File: .github/workflows/build-linux-installer.yml [hunk 3]
```diff
@@ -100,10 +100,10 @@ jobs:
       with:
         node-version: '12.x'
 
-    - name: Build .deb and .rpm packages
+    - name: Build .deb package
       run: |
         . ./activate
-        sudo apt-get -y install rpm
+        sudo apt-get -y install
```
Comment (cmmarslender) [inline .github/workflows/build-linux-installer.yml:106]: Installing nothing?

File: build_scripts/build_linux.sh [hunk 4]
```diff
@@ -27,8 +26,7 @@ echo "Chia Installer Version is: $CHIA_INSTALLER_VERSION"
 echo "Installing npm and electron packagers"
 npm install electron-packager -g
 npm install electron-installer-debian -g
-npm install electron-installer-redhat -g
-
+s
```
Comment (cmmarslender) [inline build_scripts/build_linux.sh:29]: s ?

File: build_scripts/build_linux.sh [hunk 5]
```diff
@@ -43,7 +41,7 @@ if [ "$LAST_EXIT_CODE" -ne 0 ]; then
 	exit $LAST_EXIT_CODE
 fi
 
-cp -r dist/daemon ../chia-blockchain-gui
+cssp -r dist/daemon ../chia-blockchain-gui
```
Comment (cmmarslender) [inline build_scripts/build_linux.sh:44]: typo?

**Review by cmmarslender — CHANGES_REQUESTED** [review]

**Review by hoffmang9 — DISMISSED** [review]
Looks good

**Review by justinengland — CHANGES_REQUESTED** [review]
Can we rename the installer.yml to Deb in the file name please?

**Review by cmmarslender — APPROVED** [review]

**Review by justinengland — APPROVED** [review]

**Pull request merged**